user-docs/documentation/docs/how-to-guides/3-migration-guides/2-babel-to-tsup.md

```diff
@@ -6,7 +6,9 @@ sidebar_position: 2
 
 ## Why?
 
-Coming soon...
+Swarmion used [Babel](https://babeljs.io/) to transpile internal library code. Babel made it possible to transform our Typescript code to plain Javascript (both CommonJS and ESModules).
+
+However, we decided to migrate to [Tsup](https://tsup.egoist.dev/). It uses [Esbuild](https://esbuild.github.io/) and is much more efficient to transpile libraries.
```

Comment:
A link to the migration PR could be useful here 😉

Reply:
Done